pkgs/by-name/sy/syzkaller/package.nix

```
@@ -0,0 +1,62 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  go,
+  ncurses,
+}:
+let
+  rev = "67d7ec0a69193b2acbb8cdb5109d5505a9128ac5";
+  commitDate = "2024-01-09";
+  version = "unstable-${commitDate}";
```

Unstable version should start with `0` if there are no tags:

```suggestion
  version = "0-unstable-${commitDate}";
```

Actually, `version` is only used once inside the derivation, so there is no used in defining it in the `let in` block. I think this is better:

```nix
let
  commitDate = "2024-01-09";
  dateString = lib.strings.concatStringsSep "-" [
    (lib.strings.replaceStrings [ "-" ] [ "" ] commitDate)
    "133700" # dummy time
  ];
in
stdenv.mkDerivation (finalAttrs: {
  pname = "syzkaller";
  version = "0-unstable-${commitDate}";
```